tests/npm-debug.log

```diff
@@ -1,46 +0,0 @@
-0 info it worked if it ends with ok
```

Comment:
should this file be under source control?

Reply:
No. Which is why I removed it.